app/services/document_upload.rb

```diff
@@ -33,20 +40,18 @@ def set_public_permission_on_document
   end
 
   def google_drive_virus_check
-    download_path = "#{uploaded.id}"
-    begin
-      drive_service.get_file(
-        uploaded.id,
-        acknowledge_abuse: false,
-        download_dest: download_path
-      )
-    rescue Google::Apis::ClientError
-      drive_service.delete_file(uploaded.id)
+    self.download_path = "#{uploaded.id}"
```

Comment:
If we just need to make sure to have an empty string instead of a `nil` value, can we go for `uploaded.id.to_s` instead of creating a new interpolated string?

That will be also faster and allocate less memory